.env

```diff
@@ -1,5 +1,5 @@
 # Substrate node config
-SUBSTRATE_URL=ws://127.0.0.1:9944
+SUBSTRATE_URL=ws://testnet.subsocial.network:9944
```

Comment:
We should delete .env file from offchain.

Reply:
Should I use same solution as on UI?